Moonscraper Chart Editor/Assets/Scripts/Game/UI/Menus/SongPropertiesPanelController.cs

```diff
@@ -320,6 +346,43 @@ public void RefreshAllAudioStreams()
         StartCoroutine(_RefreshAllAudioStreams());
     }
 
+    public void GetInstrumentAudioByFileNames()
```

Comment:
This function name should be more specific, would suggest something like "LoadCloneHeroAudioFromDirectory".